netsim/templates/provider/libvirt/frr-domain.j2

```diff
@@ -1,15 +1,6 @@
 {% if _vagrant_scripts.frr is not defined %}
 {%   set _vagrant_scripts.frr = True %}
 
-    $frr_script = <<-SCRIPT
-echo "Setting password for user Vagrant"
-echo vagrant:{{ n.ansible_ssh_pass }} | chpasswd
```

Comment:
Instead of deleting this, can't we do something like
```
{% if n.ansible_ssh_pass is defined %}
...
```

I think we should keep the option open to use plain passwords, if users prefer